oak/proto/storage.proto

```diff
@@ -77,7 +77,9 @@ message RollbackResponse {
 message StorageOperationRequest {
   // The name of the storage on which to perform the operation.
```

Comment:
I still think this is quite vague, what is a "storage" and what defines its name? Maybe this should point to some high level comment? Or this proto message should contain more information on how the storage works? e.g. "a storage backend is subdivided into different namespaces, whose values are UUIDs derived from a user-provided string... if multiple mutually distrusting clients end up choosing the same name then $X happens" etc. . Maybe this is already present in the code, so you can just add a link to it?